### PR TITLE
[12.0][MIG] stock_picking_back2draft

### PR DESCRIPTION
**Pull request by robyf70**
Just migrated the module to v12

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
Please preserve commit history: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#technical-method-to-migrate-a-module-from-110-to-120-branch

Also your review in v11 would be much appreciated to move things faster: https://github.com/OCA/stock-logistics-workflow/pull/550

**Comment by robyf70** [comment]
> Please preserve commit history: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#technical-method-to-migrate-a-module-from-110-to-120-branch

Ok! I'm going to push it

> 
> Also your review in v11 would be much appreciated to move things faster: #550

Sure! Will do right now

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Comment by HaraldPanten** [comment]
Hi @rousseldenis I think this PR is on the same status as https://github.com/OCA/stock-logistics-workflow/pull/550

**Review by rousseldenis — APPROVED** [review]

File: stock_picking_back2draft/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Pickings back to draft",
+    "summary": "Reopen cancelled pickings",
+    "version": "12.0.1.0.0",
+    "category": "Warehouse Management",
+    "website": "https://www.agilebg.com",
```
Comment (AaronHForgeFlow) [inline stock_picking_back2draft/__manifest__.py:8]: `'website': 'https://github.com/OCA/stock-logistics-workflow',`

**Pull request merged**